specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -24,6 +24,22 @@ To see additional help and options, run:
 
 These are the global settings for the Machine Learning Services API.
 
+``` yaml
+openapi-type: arm
+tag: package-2020-04-01
+```
```

Comment:
Well we should not adding a new snippet here, instead we need to modify the existing tag to `package-2020-04-01`.
Therefore could we please remove this, and change the `package-2020-03-01` to `package-2020-04-01` below?